ctrlf.el

```diff
@@ -129,7 +130,15 @@ active in the minibuffer during a search."
     ([remap scroll-up-command]              . ctrlf-next-page)
     ([remap scroll-down-command]            . ctrlf-previous-page)
     ([remap recenter-top-bottom]            . ctrlf-recenter-top-bottom)
+    ;; Remaps `isearch-occur' which is in another map.
+    ("M-s o"     . ctrlf-occur)
```

Comment:
I wish I could actually just remap these, but they are only temporarily bound when an isearch is active.

Reply:
So, `M-s o` is bound globally to `occur`. But in an `isearch`, `M-s o` is transiently rebound to `isearch-occur`. An `M-s C-h` here is misleading because even if you've started an isearch, it pulls up the global bindings.

I found `which-key` really helpful here because it showed the transient bindings, and I otherwise referred to the [manual](https://www.gnu.org/software/emacs/manual/html_node/emacs/Special-Isearch.html).

However, `ctrlf-forward-symbol-at-point` I goofed on. We're already defining it globally (as does isearch) and it doesn't need to be rebound again here. The rest of these, however, are only bound during `isearch`.

Note that like `M-s _` is slightly tricky because outside of the search it should be `ctrlf-forward-symbol` but inside a search it should be `ctrlf-toggle-symbol`.